.sandbox/provision/lifecycle-asg/on-resume.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# : ${ASG_NAME:=test-windows-provision}
+# : ${AVAILABILITY_ZONE:=us-east-2a}
+# : ${EC2_SSH_KEY_FILE:=/run/sandbox/fs/secrets/shared/sandbox-shared.pem}
+
+set -ex
+
+source ./common.sh
+# redirect stdout to stderr to ensure the stdout output is the desired JSON object.
+redirect_stdout
+
+validate_asg "$ASG_NAME"
+validate_az "$AVAILABILITY_ZONE"
+validate_ssh_key "$EC2_SSH_KEY_FILE"
+
+VOLUME_ID="$(get_volume_id)"
```

Comment:
Here's something bad with `set -e`. If `get_volume_id` failed, we should print more detailed message to help user understanding what's going on. Note: the users see this message are not org-admin, so they have no context about this script and how the sandbox is working. More user friendly messages are essential when surfacing errors.

Reply:
`get_volume_id` is actually calling `aws ec2 ...`. There might be different error codes that means different errors. I suppose the ideal solution is to display corresponding info to different errors. But thing is, is that really necessary given aws' normal stderr is full of information already? 

In another word, I think `aws`'s output is user friendly enough and we don't have to wrap it up again. User might search for the original error messages to understand what is happening but would get nothing if we wrap the error messages in our own way.